### PR TITLE
Protocol Requirements

### PR DESCRIPTION
**Pull request by joinimran**
Addition of protocols requirements for Windows Autopilot deployment.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5438

### CODE REVIEW

**Comment by PRMerger16** [comment]
@joinimran : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by greg-lindsay** [comment]
The sentence added is not grammatically correct. I'm not sure of the meaning. Can you please rewrite this, or explain.

File: memdocs/autopilot/software-requirements.md [hunk 1]
```diff
@@ -49,9 +49,13 @@ Windows Autopilot depends on specific features available in Windows 10, Azure Ac
   - Windows 10 Education
   - Windows 10 Enterprise 2019 LTSC
 
+### Protocols
+
+- Azure Active Directory (AAD) required both WS-Trust and WS-Federation protocol.
```
Comment (JohanFreelancer9) [inline memdocs/autopilot/software-requirements.md:54]:
```suggestion
- Azure Active Directory (AAD) requires both WS-Trust and WS-Federation protocols.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - grammar correction, not sure about technical accuracy.

**Comment by ktoliver** [comment]
@greg-lindsay Could you take another look at the proposed update to your article? 

Thanks.

**Comment by greg-lindsay** [comment]
Now we have a grammatically correct statement, but I'm not sure that it is technically correct.  And, there is no context.  Why are we making a statement about protocols here and how does this affect customers? What must they do to fulfill the requirement? 

I'm not an expert here, but I think that applications use the protocols that are mentioned and AAD provides them, or that AAD is compliant with the specifications for these protocols.  We do not say that AAD "requires" these protocols.

We need more information from the original author here.

**Comment by PRMerger6** [comment]
@joinimran : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by greg-lindsay** [comment]
No response to my questions, so closing this.  We can reopen if there is an update.

**Pull request closed without merging**